apps/test/unit/lib/jsdebugger/JsDebuggerTest.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import sinon from 'sinon';
+import {Provider} from 'react-redux';
+import {mount} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import JsDebugger, {UnconnectedJsDebugger} from '@cdo/apps/lib/jsdebugger/JsDebugger';
+import JsDebuggerUi from '@cdo/apps/lib/jsdebugger/JsDebuggerUi';
+import {getStore, registerReducers, stubRedux, restoreRedux} from '@cdo/apps/redux';
+import commonReducers from '@cdo/apps/redux/commonReducers';
+import {setPageConstants} from '@cdo/apps/redux/pageConstants';
+
+describe('The JSDebugger component', () => {
+  let root, jsDebugger;
+
+  beforeEach(() => {
+    stubRedux();
+    registerReducers(commonReducers);
+
+    const codeTextbox = document.createElement('div');
+    document.body.appendChild(codeTextbox);
+    codeTextbox.id = 'codeTextbox';
```

Comment:
Should we remove this from the DOM after each? Right now you're adding one per test - is the component always going to use the most recently added one, or will it use an element from a previous test?